stellar-multitenant/pkg/internal/validators/tenant_validator.go

```diff
@@ -104,12 +104,22 @@ func (tv *TenantValidator) ValidateUpdateTenantRequest(reqBody *UpdateTenantRequ
 		if _, err = url.ParseRequestURI(*reqBody.BaseURL); err != nil {
 			tv.Check(false, "base_url", "invalid base URL value")
 		}
+
+		*reqBody.BaseURL, err = utils.SetURLSchemeIfEmpty(*reqBody.BaseURL)
+		if err != nil {
+			tv.Check(false, "base_url", "invalid base URL value. Verify the URL protocol scheme")
```

Comment:
This extra check would override the previous one, right? I believe you should only apply the second `"base_url"` check if the first one has not returned an error.